packages/wrangler/src/cloudchamber/cli/locations.ts

```diff
@@ -19,7 +19,13 @@ export async function getLocation(
 	args: { location?: string },
 	options: { skipLocation?: boolean } = {}
 ): Promise<LocationID | "Skip"> {
+	const { external_account_id } = await loadAccount();
 	const locations = await getLocations();
+
+	if (locations.length === 0) {
+		throw new Error(`No allowed locations found for account ${external_account_id}`);
+	}
+
```

Comment:
Let's just remove `getLocations` completely, all it does is return the `locations` property from the account. So this would instead become:

```suggestion
	const account = await loadAccount();
	const locations = account.locations;

	if (locations.length === 0) {
		throw new Error(`No allowed locations found for account ${account.external_account_id}`);
	}

```

Reply:
I did originally do something like this but then types got weird with `CompleteAccountLocationCustomer` vs. `Location` somehow.

Lemme give it another go